src/OpenDiffix.Core.Tests/Aggregator.Tests.fs

```diff
@@ -49,16 +49,18 @@ let makeStandardArgs hasValueArg random length =
 /// Verifies that merging 2 separately aggregated sequences is equivalent
 /// to a single aggregation of those 2 sequences concatenated.
 let ensureConsistentMerging ctx fn sourceArgs destinationArgs =
-  let sourceAggregator = Aggregator.create fn
+  let DUMMY_ARGS = []
+
+  let sourceAggregator = Aggregator.create (fn, DUMMY_ARGS)
   sourceArgs |> List.iter sourceAggregator.Transition
 
-  let destinationAggregator = Aggregator.create fn
+  let destinationAggregator = Aggregator.create (fn, DUMMY_ARGS)
```

Comment:
Ha! I've got almost the same thing, I just didn't come across this trick. You can close this, I'll borrow the DUMMY_ARGS part

Reply:
And thank you :bow: